src/test/java/com/iota/iri/controllers/TransactionViewModelTest.java

```diff
@@ -396,47 +390,40 @@ public void testManyTXInDB() throws Exception {
+            trits[i] = (byte) (seed.nextInt(3) - 1);
+        }
+
         transaction.bytes = Converter.allocateBytesForTrits(trits.length);
         Converter.bytes(trits, 0, transaction.bytes, 0, trits.length);
         return transaction;
     }
-    public static int[] getRandomTransactionWithTrunkAndBranch(Hash trunk, Hash branch) {
-        int[] trits = getRandomTransactionTrits();
+    public static byte[] getRandomTransactionWithTrunkAndBranch(Hash trunk, Hash branch) {
+        byte[] trits = getRandomTransactionTrits();
         System.arraycopy(trunk.trits(), 0, trits, TransactionViewModel.TRUNK_TRANSACTION_TRINARY_OFFSET,
                 TransactionViewModel.TRUNK_TRANSACTION_TRINARY_SIZE);
         System.arraycopy(branch.trits(), 0, trits, TransactionViewModel.BRANCH_TRANSACTION_TRINARY_OFFSET,
                 TransactionViewModel.BRANCH_TRANSACTION_TRINARY_SIZE);
         return trits;
     }
-    public static int[] getRandomTransactionWithTrunkAndBranchValidBundle(Hash trunk, Hash branch) {
-        int[] trits = getRandomTransactionTrits();
-        System.arraycopy(trunk.trits(), 0, trits, TransactionViewModel.TRUNK_TRANSACTION_TRINARY_OFFSET,
-                TransactionViewModel.TRUNK_TRANSACTION_TRINARY_SIZE);
-        System.arraycopy(branch.trits(), 0, trits, TransactionViewModel.BRANCH_TRANSACTION_TRINARY_OFFSET,
-                TransactionViewModel.BRANCH_TRANSACTION_TRINARY_SIZE);
-        System.arraycopy(Hash.NULL_HASH.trits(), 0, trits, TransactionViewModel.CURRENT_INDEX_TRINARY_OFFSET,
-                TransactionViewModel.CURRENT_INDEX_TRINARY_SIZE);
-        System.arraycopy(Hash.NULL_HASH.trits(), 0, trits, TransactionViewModel.LAST_INDEX_TRINARY_OFFSET,
-                TransactionViewModel.LAST_INDEX_TRINARY_SIZE);
-        System.arraycopy(Hash.NULL_HASH.trits(), 0, trits, TransactionViewModel.VALUE_TRINARY_OFFSET,
-                TransactionViewModel.VALUE_TRINARY_SIZE);
-
-        final Sponge curlInstance = SpongeFactory.create(SpongeFactory.Mode.KERL);
-        final int[] bundleHashTrits = new int[TransactionViewModel.BUNDLE_TRINARY_SIZE];
-        curlInstance.reset();
-        curlInstance.absorb(trits, TransactionViewModel.ESSENCE_TRINARY_OFFSET, TransactionViewModel.ESSENCE_TRINARY_SIZE);
-        curlInstance.squeeze(bundleHashTrits, 0, bundleHashTrits.length);
-
-        System.arraycopy(bundleHashTrits, 0, trits, TransactionViewModel.BUNDLE_TRINARY_OFFSET,
-                TransactionViewModel.BUNDLE_TRINARY_SIZE);
+    public static byte[] getRandomTransactionTrits() {
+        byte[] out = new byte[TransactionViewModel.TRINARY_SIZE];
 
-        return trits;
-    }
-    public static int[] getRandomTransactionTrits() {
-        return Arrays.stream(new int[TransactionViewModel.TRINARY_SIZE]).map(i -> seed.nextInt(3)-1).toArray();
+        for(int i = 0; i < out.length; i++) {
+            out[i] = (byte) (seed.nextInt(3) - 1);
+        }
+
+        return out;
     }
```

Comment:
I agree that this is more reasdable than the one liner.
Another option was to not change the code and just add new lines before each aggregate call.

Anyhows, both variations are fine with me.